config/packages/test/fixtures.yaml

```diff
@@ -0,0 +1,69 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+sylius_fixtures:
+    suites:
+        test:
```

Comment:
I'm not convinced about this configuration, just to check something. I would vote for adding these attributes to default Sylius fixtures file + write PHPUnit test for ProductExampleFactory

Reply:
I just wanted to be 100% that both `ProductExampleFactory` works fine... On the other hand, to solve the issue indeed the test for only `ProductExampleFactory` would be sufficient 👍